mono/mini/method-to-ir.c

```diff
@@ -3812,18 +3814,31 @@ mono_method_check_inlining (MonoCompile *cfg, MonoMethod *method)
 		if ((inlinelimit = g_getenv ("MONO_INLINELIMIT"))) {
 			inline_limit = atoi (inlinelimit);
 			llvm_jit_inline_limit = inline_limit;
+			llvm_aot_inline_limit = inline_limit;
 			g_free (inlinelimit);
 		} else {
 			inline_limit = INLINE_LENGTH_LIMIT;
 			llvm_jit_inline_limit = LLVM_JIT_INLINE_LENGTH_LIMIT;
+			llvm_aot_inline_limit = LLVM_AOT_INLINE_LENGTH_LIMIT;
 		}
 		inline_limit_inited = TRUE;
 	}
 
+#ifdef ENABLE_NETCORE
```

Comment:
I would rather `ifdef` the definition of `LLVM_AOT_INLINE_LENGTH_LIMIT` than duplicating this code.

Reply:
This is not really duplicate code, its more like #ifdef <future> #else <past>. The purpose is to not break existing code.

Reply:
it's kind of worse, because `llvm_aot_inline_limit` is ignored for `!ENABLE_NETCORE`, so its existence is misleading.

I'm proposing to use the block in `ENABLE_NETCORE` for everything and instead define `LLVM_AOT_INLINE_LENGTH_LIMIT` like this:

```c
#ifdef ENABLE_NETCORE
#define LLVM_AOT_INLINE_LENGTH_LIMIT 30
#else
#define LLVM_AOT_INLINE_LENGTH_LIMIT 20
#endif
```

Reply:
Thats not the same, the old code was only using the llvm limit for llvm+!aot.

Reply:
yeah, and otherwise it would fallback to `INLINE_LENGTH_LIMIT`? to make it more obvious, maybe this:

```c
#ifdef ENABLE_NETCORE
#define LLVM_AOT_INLINE_LENGTH_LIMIT 30
#else
#define LLVM_AOT_INLINE_LENGTH_LIMIT INLINE_LENGTH_LIMIT
#endif
```
anyway, it doesn't really matter.